test/unit/ngsiv2/httpBindings-test.js

```diff
@@ -0,0 +1,740 @@
+        var getOptions = {
+            url: 'http://localhost:' + config.http.port + '/iot/d',
+            method: 'POST',
+            qs: {
+                i: 'HTTP_2',
+                k: '1234'
+            },
+            headers: {
+                'Content-type': 'text/plain'
+            },
+            body: 'temperature|23|humidity|98#temperature|16|humidity|34'
+        };
+
+        beforeEach(function() {
+            contextBrokerMock
+            .matchHeader('fiware-service', 'smartGondor')
+            .matchHeader('fiware-servicepath', '/gardens')
+            .post('/v2/entities/Second%20UL%20Device/attrs',
+                utils.readExampleFile('./test/unit/ngsiv2/contextRequests/multipleMeasure.json'))
+            .reply(204);
+
+            contextBrokerMock
+            .matchHeader('fiware-service', 'smartGondor')
+            .matchHeader('fiware-servicepath', '/gardens')
+            .post('/v2/entities/Second%20UL%20Device/attrs',
+                    utils.readExampleFile('./test/unit/ngsiv2/contextRequests/secondMultipleMeasure.json'))
+            .reply(204);
+            });
+
+        it('should end up with a 200OK status code', function(done) {
+            request(getOptions, function(error, response, body) {
+                should.not.exist(error);
+                response.statusCode.should.equal(200);
+                done();
+            });
+        });
+        it('should send a two update context requests to the Context Broker one with each attribute', function(done) {
+            request(getOptions, function(error, response, body) {
+                contextBrokerMock.done();
+                done();
+            });
+        });
+    });
+
+    describe('When a request arrives to the IoT Agent without Content-type header', function() {
+        var postOptions = {
+            url: 'http://localhost:' + config.http.port + '/iot/d',
+            method: 'POST',
+            qs: {
+                i: 'urn:x-iot:smartsantander:u7jcfa:fixed:t311',
+                k: '1234',
+                t: '2016-05-11T10:12:26.476659Z'
+            },
+            body: 'bat|75.0'
+        },
+        provisionOptions = {
+            url: 'http://localhost:' + config.iota.server.port + '/iot/devices',
+            method: 'POST',
+            json: utils.readExampleFile('./test/deviceProvisioning/provisionDeviceProduction.json'),
+            headers: {
+                'fiware-service': 'smartGondor',
+                'fiware-servicepath': '/gardens'
+            }
+        };
+
+        beforeEach(function(done) {
+            nock.cleanAll();
+            // Note: /v1/updateContext response is not processed by IOTA so its content is irrelevant,
+            // as far as it is a 200 OK
+            contextBrokerMock = nock('http://192.168.1.1:1026')
+            .post('/v1/updateContext')
+            .reply(200 , {})
+            .post('/v2/entities/urn:x-iot:smartsantander:u7jcfa:fixed:t311/attrs')
```

Comment:
The payload for this post should be defined in the mock, as in other test cases.

(I think this was already mentioned in the first review, but maybe it get lost... :)

Reply:
Fixed in bd211fb8e94eb003fe88745cf2ef45b83f0b2f9f